contest/views.py

```diff
@@ -233,6 +233,8 @@ def ask(request):
             form = ClarificationForm(request.POST)
             if form.is_valid():
                 new_clarification = form.save()
+                new_clarification.reply = ''
+                new_clarification.save()
```

Comment:
Is it possible to add a button "No Response" and press it will fill "No Response. Please read the problem statement" into reply content?